modules/vaos/app/controllers/vaos/appointment_requests_controller.rb

```diff
@@ -5,16 +5,23 @@ class AppointmentRequestsController < VAOS::BaseController
     before_action :validate_params, only: :index
 
     BASE_REQUIRED_PARAMS = %i[
-      option_date1 option_time1 option_date2 option_time2 option_date3 option_time3 status
-      appointment_type visit_type phone_number facility
+      option_date1 option_time1 option_date2 option_time2 option_date3 option_time3 appointment_type visit_type
+      phone_number facility
     ].freeze
 
     BASE_PARAMS_WHITELIST = [
-      :option_date1, :option_time1, :option_date2, :option_time2, :option_date3, :option_time3, :status,
+      :type, :option_date1, :option_time1, :option_date2, :option_time2, :option_date3, :option_time3, :status,
       :appointment_type, :visit_type, :phone_number, :email, :purpose_of_visit, :other_purpose_of_visit,
       :provider_id, :provider_name, :second_request, :second_request_submitted, :requested_phone_call, :type_of_care_id,
-      best_timeto_call: [], facility: %i[name facility_code state city parent_site_code object_type],
-      appointment_request_detail_code: [], patient: %i[inpatient text_messaging_allowed]
+      :additional_information, :address, :city, :state, :zip_code, :distance_willing_to_travel, :new_message,
+      :office_hours, :preferred_city, :preferred_state, :preferred_zip_code, :provider_option, :preferred_language,
```

Comment:
Office hours here is a string -- the attributes in the form is an array of strings. If that's correct, no problem. Just an FYI.

Reply:
Yes, there is a lot of redundancy, but garbage in, garbage out. 🤷‍♂